ui/eureka/eureka.js

```diff
@@ -1832,15 +2273,18 @@ class EurekaTracker {
         aspect.classList.remove(aspect.classList.item(0));
       aspect.classList.add('aspect-ratio-' + this.zoneInfo.shortName);
 
-      document.getElementById('map-image').src = this.zoneInfo.mapImage;
+      if (this.zoneInfo.mapImage) {
+        document.getElementById('map-image').src = this.zoneInfo.mapImage;
+        window.clearInterval(this.updateTimesHandle);
+        this.updateTimesHandle = window.setInterval((function() {
+          this.UpdateTimes();
+        }).bind(this), this.options.RefreshRateMs);
```

Comment:
perfect use-case for an arrow function?
```suggestion
        this.updateTimesHandle = window.setInterval(
            () => void this.UpdateTimes(),
            this.options.RefreshRateMs
        );
```